### DISCUSSION TITLE
Per-Camera Retention Settings

### DISCUSSION THREAD

**Discussion by JonLaliberte** [ep1cman/unifi-protect-backup#173] [Ideas]
I have some cameras I want to keep indefinitely (I use a 100y setting for this), and others that could get scrubbed after a few weeks.

It would be nice if there were a way to set retention per camera.

Not sure if this is difficult because of dependence on rclone though. I haven't had a chance to look at the code at all for this.

**Comment by ep1cman** [comment]
The way the "purge" works is per file, and the database does contain the camera id for each file so it could definitely be implemented
